### ISSUE TITLE
[a11y]: FileUploadDropContainer issue with aria-required

### ISSUE THREAD

**Issue by AustinGitHub** [carbon-design-system/carbon#19067]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

1.78.2

### React version

17.0.2

### Automated testing tool and ruleset

IBM Accessibility Tool

### Description

We are unable to properly add a `aria-required` as it complains about a role
```
<div aria-required="true" className={styles.uploader}>
          <FileUploaderDropContainer
            accept={[".csv"]}
            multiple
            labelText={intl.formatMessage(messages.dragDropFile)}
            onAddFiles={onAddFiles}
            size="lg"
          />
        </div>
```

This is the error we see but we can't properly add role due to how carbon renders this component and we can't pass through any props to address it properly. 

![Image](https://github.com/user-attachments/assets/52ddc107-a544-4eb7-a5fa-fa1282f9e4cb)

Thanks!

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-fileuploader--file-uploader-drop-container

### Steps to reproduce

Hard to show because the storybook has a ton of violations on this page so not sure how to provide reproduction

https://react.carbondesignsystem.com/?path=/story/components-fileuploader--file-uploader-drop-container

### Suggested Severity

Severity 3 = User can complete task, and/or has a workaround within the user experience of a given component.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems
